src/expressions/functions/builtInFunctions.string.ts

```diff
@@ -694,7 +694,7 @@ export default {
 		{
 			namespaceURI: FUNCTIONS_NAMESPACE_URI,
 			localName: 'string-join',
-			argumentTypes: ['xs:string*'],
+			argumentTypes: ['xs:anyAtomicType*'],
```

Comment:
this here

Reply:
Could you also move this function declaration so it's besides the `string-join` one with 2 arguments? Every other overload seems to be next to one another except this one.
(Or move the other one of course, doesn't really matter)